ravenframework/SupervisedLearning/DynamicModeDecompositionControl.py

```diff
@@ -397,6 +478,66 @@ def writeXML(self, writeTo, targets = None, skip = None):
                    "matrixShape":",".join(str(x) for x in np.shape(self.__Ctilde[smp, :, :]))}
         writeTo.addVector("Ctilde","realization",valDict, root=targNode, attrs=attributeDict)
 
+  def getSolutionMetadata(self):
+    """
+      Get solution metadata
+    """
+    solutionMetadata = {}
+
+
+    targNode = writeTo._findTarget(writeTo.getRoot(), target)
```

Comment:
It seems writeTo is not pre-defined or provided through function input, could you check it?

Reply:
it is an input to the function
```writeXML(self, writeTo, targets = None, skip = None)```

Reply:
ops. removed